### ISSUE TITLE
[Small bug, I think] The lich cloak is transferred to another character, can then not be dropped, but frees space for the lich-transformed character to wear another cloak

### ISSUE THREAD

**Issue by rubyFeedback** [LCA-EET/Lichdom#18]
TL;DR: The lich cloak is transferred to another character, can then not be dropped, but frees space for the lich-transformed character to wear another cloak

So in my new party, where I am also testing the latest lichdom mod, I think I ran into a small bug.

Edwin became a lich. This occupies one slot for the lich cloak.

However had, I wanted to solve Neera's quest next, so I dropped Edwin. To pick him up again at a later time.

When I kicked Edwin out, the lich-cloak was transferred into my main character and I could not remove it. It
just stays in the inventory now, similar to a familiar that is put into the inventory, or Hexxat's cloak I think
which is a bit buggy too at times.

Now I finished Neera's quest and I put Edwin back into the party. Interestingly I can not transfer the
lich cloak to him, it just stays with the main character, and I can have Edwin wear another cloak.

So I think this is a slight bug.

Possibly aside from this bug, perhaps the character that became, should get some essence or container 
that a lich may normally have instead, like a phylactum or what's the name, rather than a cloak. But
either way, the bug is that the cloak comes off; I think it should stay with Edwin or whoever becomes
a lich, at all times.

**Comment by LCA-EET** [comment]
Thanks for reporting the issue. It sounds like the lich cloak is being treated as a Critical Item when it shouldn't be. I'll investigate and issue a fix this weekend.

**Comment by LCA-EET** [comment]
This is fixed in the current release (1.2).
